README.md

```diff
@@ -16,6 +16,32 @@ It is also possible to whitelist all commits in a PR with the `needs-ci` tag.
 However, this should be only used with trusted submitters since they can still
 change the commits in the PR.
 
+## Git workflow
+
+1. Merge pull requests to master without merge commits to keep a less
```

Comment:
There are branch protection rules in GitHub, I enabled:

"Require linear history
Prevent merge commits from being pushed to matching branches."

This should take care of this as far as I understand and also allow to use the web interface instead of git commands.

Reply:
Also adjusted the setting for the merge button to only allow rebase. The branch protection rule should probably also be added for `production`.

Reply:
@tyll, thanks, will it perform a fast-forward merge or do a rebase resulting in a new commit?

Reply:
should be a fast-forward merge